integrations/arize.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: integration
+name: Arize AI
```

Comment:
Do you think it's possible to just split between Arize and Arize Phoenix? I know that's a bit confusing but one is fully-OSS and so it might be a nice option for developers to check out in isolation.